Header header

int16 thumb_2
int16 thumb_1
int16 index_2
int16 index_1
int16 middle_2
int16 middle_1
int16 ring_2
int16 ring_1
int16 little_2
int16 little_1
int16 palm_arch
# int16 nop
int16 thumb_crossover
# int16 nop2 #nop to be deleted
int16 press_thumb
int16 press_index
int16 press_middle
int16 press_ring
int16 press_little
int16 abd_thumb
int16 abd_index
int16 abd_ring
int16 abd_little 

